.gitignore

```diff
@@ -34,3 +34,4 @@ results.xml
 .pytest_cache/
 pyvenv.cfg
 junit
+/.vs
```

Comment:
What is under `/.vs`, I am surprised you need this

Reply:
I don't need it but VS Code creates the folder.

Reply:
the forward slash here implies its from the root of the disk.. did you just mean .vs without the leading slash? also should probably go into .dockerignore